src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/worker/apis/fleet/data_helper.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export interface AgentPolicyCreateBody {
+  advanced_settings?: Record<string, any>;
+  agent_features?: Array<{ enabled: boolean; name: string }>;
+  agentless?: {
+    cloud_connectors?: { enabled: boolean; target_csp?: string };
+    resources?: { requests?: { cpu?: string; memory?: string } };
+  };
+  data_output_id?: string;
+  description?: string;
```

Comment:
please move type to separate file `./types` similar to https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/worker/apis/cases/types.ts